docker-compose-generator/docker-fragments/opt-add-electumx.yml

```diff
@@ -5,12 +5,13 @@ services:
     environment:
       BITCOIN_EXTRA_ARGS: |
         txindex=1
+        rpcauth=electrumx:7d8ee47c089e6072635f82b34796e878$13649d99453ccdf78e85007286422599c00e6953289f41bf8e92961076ba14db
```

Comment:
you need to put an additional `$` (ie, `$$`) because the $ is interpreted by the yaml, you must escape it.